pkg/stacks/templates/engines/util.go

```diff
@@ -0,0 +1,41 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package engines
```

Comment:
https://dmitri.shuralyov.com/idiomatic-go#use-singular-form-for-collection-repo-folder-name

`engine` may be better.  We haven't followed this practice in Crossplane very well.
Go has exceptions to this practice in cases where the package name would conflict with a reserved word (strings, bytes, errors), so I'm just offering this up for consideration.

Reply:
Good suggestion! I didn't know about this. Since we haven't followed the practice consistently in the codebase, I'm going to leave this one for now, but I'll think about it some more for future changes.